packages/frontend/editor-ui/src/App.vue

```diff
@@ -149,6 +169,13 @@ useExposeCssVar('--ask-assistant-floating-button-bottom-offset', askAiFloatingBu
 			<div :id="APP_MODALS_ELEMENT_ID" :class="$style.modals">
 				<Modals />
 			</div>
+
+			<N8nCommandBar
+				v-if="showCommandBar"
```

Comment:
nit: based on the name of this variable one might get an idea that this would be true when the bar is visible, not when it just can be toggled on